art/estimators/object_detection/pytorch_object_detector.py

```diff
@@ -247,8 +249,41 @@ def _preprocess_and_convert_inputs(
 
         return x_preprocessed, y_preprocessed
 
+    def _translate_labels(self, labels: List[Dict[str, "torch.Tensor"]]) -> Any:
+        """
+        Translate object detection labels from ART format (torchvision) to the model format (torchvision) and
+        move tensors to GPU, if applicable.
+
+        :param labels: Object detection labels in format x1y1x2y2 (torchvision).
+        :return: Object detection labels in format x1y1x2y2 (torchvision).
+        """
+        labels_translated = [{k: v.to(self.device) for k, v in y_i.items()} for y_i in labels]
+        return labels_translated
+
+    def _translate_predictions(self, predictions: Any) -> List[Dict[str, np.ndarray]]:  # pylint: disable=R0201
```

Comment:
Can we remove the pylint comment and decorate this function with `@staticmethod`?

Reply:
Since `_translate_predictions` needs to be overridden for all subclasses, it cannot be a static method. For the specific cases of Faster RCNN, it does not use any of the properties, but this is not the case for YOLO and Detr.